BaseListener/build.cmd

```diff
@@ -0,0 +1,2 @@
+dotnet restore
+dotnet lambda package --configuration release --framework netcoreapp3.1 --output-package bin/release/netcoreapp3.1/base-listener.zip
```

Comment:
No.
I have modified the Renamer.ps1 file so that it is called with an additional parameter which replaces all instances of `base-listener`. See the readme in the root of the repo for instructions.